go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/datumbrain/nulltypes
+
+go 1.15
```

Comment:
Does it require any third party package?

Reply:
No. It only uses `database/sql/driver`, `encoding/json` and `time`.